### ISSUE TITLE
print.survHE() fails when no covariates are supplied to a generalized gamma model using HMC

### ISSUE THREAD

**Issue by andbe** [giabaio/survHE#26]
MRE:
```
dat = survival::ovarian
model = survHE::fit.models(
  formula = Surv(futime, fustat) ~ 1,
  data = dat,
  distr = "gengamma",
  method = "hmc"
)
print(model)
```

returns `Error in table[grep("beta", rownames(table)), ][1, ] incorrect number of dimensions`

**Comment by andbe** [comment]
Fix proposed in #27

**Comment by giabaio** [comment]
Thanks --- accepted and merged in the current version!
